git-clang-format.py

```diff
@@ -79,14 +79,18 @@ def main():
       'c', 'h',  # C
       'm',  # ObjC
       'mm',  # ObjC++
-      'cc', 'cp', 'cpp', 'c++', 'cxx', 'hh', 'hpp', 'hxx', 'inc', 'inl',  # C++
-      'cu',  # CUDA
+      'cc', 'cp', 'cpp', 'c++', 'cxx', 'hh', 'hpp', 'hxx', 'inc',  # C++
+      'ccm', 'cppm', 'cxxm', 'c++m',  # C++ Modules
```

Comment:
Any reason to remove the `inl`  extension?

Reply:
Note: **asio** use `ipp`, **boost**, and **ACE** use `inl`

Reply:
I've re-added the inl file ending and additionally added ipp.